models/examples/example-1-repeat-dispensing/ConvertResponse-Hl7V3Message.xml

```diff
@@ -43,7 +43,7 @@
         <id root="F68CEA42-E8A0-D799-C874-163048F0D390"/>
         <code code="163501000000109" codeSystem="2.16.840.1.113883.2.1.3.2.4.15" displayName="Prescription - FocusActOrEvent (record artifact)"/>
         <effectiveTime value="20200610120600"/>
-        <typeId extension="PORX_MT132004UK31" root="2.16.840.1.113883.2.1.3.2.4.18.7"/>
+        <typeId extension="PORX_MT132004SM31" root="2.16.840.1.113883.2.1.3.2.4.18.7"/>
```

Comment:
Have these been updated correctly?

Reply:
These are supposed to be the UK31 IDs, i double checked with the spine devs